source/server/http/admin.cc

```diff
@@ -150,6 +150,16 @@ Http::FilterTrailersStatus AdminFilter::decodeTrailers(Http::HeaderMap&) {
   return Http::FilterTrailersStatus::StopIteration;
 }
 
+void AdminFilter::onDestroy() {
+  for (const auto& callback : on_destroy_callbacks_) {
```

Comment:
for the record, generally we shouldn't use 'auto' as the type of a range variable when looping over a simple (non-map) container.  See

https://google.github.io/styleguide/cppguide.html#auto

However in this case it's OK because the usage makes it pretty obvious what you've got.